keyboards/reviung41/keymaps/default/rules.mk

```diff
@@ -0,0 +1 @@
+RGBLIGHT_ENABLE = yes
```

Comment:
This should be configured at the keyboard level in `keyboards/reviung41/rules.mk`. Please delete this file:

```
git rm -- keyboards/reviung41/keymaps/default/rules.mk
```